app/uk/gov/hmrc/cgtpropertydisposals/models/returns/CalculateCgtTaxDueRequest.scala

```diff
@@ -31,7 +31,8 @@ final case class CalculateCgtTaxDueRequest(
   reliefDetails: CompleteReliefDetailsAnswers,
   exemptionAndLosses: CompleteExemptionAndLossesAnswers,
   estimatedIncome: AmountInPence,
-  personalAllowance: AmountInPence
+  personalAllowance: AmountInPence,
+  initialGainOrLoss: Option[AmountInPenceWithSource]
```

Comment:
this could probably be just `AmountInPence` - when requesting the calculation, that field should always be a user supplied value

Reply:
done